samples/client/petstore/typescript-angular-v2/default/model/ApiResponse.ts

```diff
@@ -23,3 +23,5 @@ export interface ApiResponse {
     message?: string;
 
 }
```

Comment:
Still wondering about those newlines

Reply:
I can try to clean it up later with a separate PR.

Reply:
Opened #6788 to address this :)